src/nni_manager/core/nnimanager.ts

```diff
@@ -135,6 +135,42 @@ class NNIManager implements Manager {
         return this.dataStore.storeTrialJobEvent('ADD_CUSTOMIZED', '', hyperParams);
     }
 
+    public async resubmitTrialJob(data: string): Promise<void> {
+        // get trail job info by jobId
+        const trialJobId = JSON.parse(data).job_id
```

Comment:
The parameter should be trial job id, it should not need to be parsed inside this function.

Reply:
fixed